src/components/views/elements/PersistentApp.tsx

```diff
@@ -33,22 +39,25 @@ interface IProps {
 interface IState {
     roomId: string;
     persistentWidgetId: string;
+    rightPanelPhase?: RightPanelPhases;
 }
 
 @replaceableComponent("views.elements.PersistentApp")
 export default class PersistentApp extends React.Component<IProps, IState> {
     private roomStoreToken: EventSubscription;
-
+    private dispatcherRef: string;
```

Comment:
please keep empty lines between blocks (between the fields and the constructor, in this case)